django_migration_vis/management/commands/visualizemigrations.py

```diff
@@ -32,7 +39,26 @@ def _create_digraph(self):
             self._add_dependencies(node)
 
     @staticmethod
-    def _style_label(tupled_node):
+    def _censor(text):
+        res = []
+        for c in text:
+            if c not in '0123456789_':
+                c = chr(random.randint(ord('a'), ord('z')))
+            res.append(c)
+        return ''.join(res)
+
+    def _censor_using_cache(self, text):
+        try:
+            return self._censor_cache[text]
+        except KeyError:
+            censored = self._censor(text)
+            self._censor_cache[text] = censored
```

Comment:
Don't you want to add the node name excluding the tail that is the combination of an underline and numbers in the `_censor_cache` dictionary so that they have consistent censored node names? I guess you need something like
```
nodeid = '_'.join(text.split('_')[:-1])
nodenum = text.split('_')[-1]
try:
     return '_'.join(self._censor_cache[nodeid], nodenum)
except KeyError:
     censored = self._censor(nodeid)
     self._censor_cache[nodeid] = censored
     return '_'.join(censored, nodenum)
```
In the case that you want this,
you can also erase the `if c not in '0123456789_':` line in the `_censor` method.

Reply:
Hmm... I realized that numbers appear not only in the tail but also beginning and middle.... then with `re.search('[_a-zA-Z]+', text).group(0)`?

Reply:
I would consider obfuscation where single words always end up as the same value as an anti-feature.

Reply:
that makes sense